modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/secret/management/v1/SecretManagementFailureTest.java

```diff
@@ -0,0 +1,205 @@
+
+        if (!StringUtils.equals(MultitenantConstants.SUPER_TENANT_DOMAIN_NAME, tenant)) {
+            String body = readResource("add-secret-type.json");
+            Response response =
+                    getResponseOfPost(SECRET_TYPE_API_BASE_PATH, body);
+
+            response.then()
+                    .log().ifValidationFails()
+                    .assertThat()
+                    .statusCode(HttpStatus.SC_CREATED)
+                    .header(HttpHeaders.LOCATION, notNullValue());
+            String location = response.getHeader(HttpHeaders.LOCATION);
+            assertNotNull(location);
+            secretTypeName = location.substring(location.lastIndexOf("/") + 1);
+            assertNotNull(secretTypeName);
+            response = getResponseOfPost(SECRET_TYPE_API_BASE_PATH, body);
+            validateErrorResponse(response, HttpStatus.SC_CONFLICT, "SECRETM_00019");
+        }
+    }
+
+    @Test
+    public void testGetSecretTypeByInvalidName() {
+
+        Response response = getResponseOfGet(SECRET_TYPE_API_BASE_PATH + PATH_SEPARATOR + "randomName");
+
+        validateErrorResponse(response, HttpStatus.SC_NOT_FOUND, "SECRETM_00021");
+
+    }
+
+    @Test
+    public void testUpdateSecretTypeByInvalidName() throws IOException {
+
+        if (!StringUtils.equals(MultitenantConstants.SUPER_TENANT_DOMAIN_NAME, tenant)) {
+            String body = readResource("update-secret-type.json");
+            Response response = getResponseOfPut(
+                    SECRET_TYPE_API_BASE_PATH + PATH_SEPARATOR + "randomName", body);
+
+            validateErrorResponse(response, HttpStatus.SC_NOT_FOUND, "SECRETM_00021");
+        }
+    }
+
+    @Test
+    public void testDeleteSecretTypeByInvalidName() {
+
+        if (!StringUtils.equals(MultitenantConstants.SUPER_TENANT_DOMAIN_NAME, tenant)) {
+            Response response = getResponseOfDelete(
+                    SECRET_TYPE_API_BASE_PATH + PATH_SEPARATOR + "randomName");
+
+            validateErrorResponse(response, HttpStatus.SC_NOT_FOUND, "SECRETM_00021");
+        }
+    }
+
+    @Test
+    public void testAddSecretConflict() throws IOException {
+
+        String body = readResource("add-secret.json");
+        Response response =
+                getResponseOfPost(SECRET_API_BASE_PATH + PATH_SEPARATOR + SECRET_TYPE, body);
+
+        response.then()
+                .log().ifValidationFails()
+                .assertThat()
+                .statusCode(HttpStatus.SC_CREATED)
+                .header(HttpHeaders.LOCATION, notNullValue());
+        String location = response.getHeader(HttpHeaders.LOCATION);
+        assertNotNull(location);
+        secretName = location.substring(location.lastIndexOf("/") + 1);
+        assertNotNull(secretName);
+        response = getResponseOfPost(SECRET_API_BASE_PATH + PATH_SEPARATOR + SECRET_TYPE, body);
+        validateErrorResponse(response, HttpStatus.SC_CONFLICT, "SECRETM_00006");
+
```

Comment:
Unnecessary newline

Reply:
Addressed with commit [252de16](https://github.com/wso2/product-is/pull/12546/commits/252de169987046a367e9c09f983731935058c944)